microsoft/testsuites/network/common.py

```diff
@@ -43,6 +53,8 @@ def initialize_nic_info(
         node_nic_info = Nics(node)
         node_nic_info.initialize()
         for _, node_nic in node_nic_info.nics.items():
+            if not node_nic.ip_addr:
```

Comment:
Add code comments to explain the new logic.

Reply:
updated